docs/guides/first-app.md

```diff
@@ -605,90 +631,141 @@ func main() {
 
-	resp, err = client.Check(ctx, &pb.CheckRequest{User: emilia, TestUserset: post1Writer})
+	resp, err = client.CheckPermission(ctx, &pb.CheckPermissionRequest{
+		Resource: firstPost,
+		Permission: "write",
+		Subject: emilia,
+	})
 	if err != nil {
 		log.Fatalf("failed to check permission: %s", err)
 	}
-	// resp.Membership == pb.CheckResponse_MEMBER
+	// resp.Permissionship == v1.CheckPermissionResponse_PERMISSIONSHIP_HAS_PERMISSION
 
-	resp, err = client.Check(ctx, &pb.CheckRequest{User: beatrice, TestUserset: post1Reader})
+	resp, err = client.CheckPermission(ctx, &pb.CheckPermissionRequest{
+		Resource: firstPost,
+		Permission: "read",
+		Subject: beatrice,
+	})
 	if err != nil {
 		log.Fatalf("failed to check permission: %s", err)
 	}
-	// resp.Membership == pb.CheckResponse_MEMBER
+	// resp.Permissionship == v1.CheckPermissionResponse_PERMISSIONSHIP_HAS_PERMISSION
 
-	resp, err = client.Check(ctx, &pb.CheckRequest{User: beatrice, TestUserset: post1Writer})
+	resp, err = client.CheckPermission(ctx, &pb.CheckPermissionRequest{
+		Resource: firstPost,
+		Permission: "write",
+		Subject: beatrice,
+	})
 	if err != nil {
 		log.Fatalf("failed to check permission: %s", err)
 	}
-	// resp.Membership != pb.CheckResponse_MEMBER
+	// resp.Permissionship == v1.CheckPermissionResponse_PERMISSIONSHIP_NO_PERMISSION
 }
 ```
 
   </TabItem>
   <TabItem value="python">
 
 ```python
-from authzed.api.v0 import CheckRequest, Client, ObjectAndRelation, User
+from authzed.api.v1 import (
+    CheckPermissionRequest,
+    CheckPermissionResponse,
+    Client,
+    ObjectReference,
+    SubjectReference,
+)
 from grpcutil import bearer_token_credentials
 
-
-emilia = User("blog/user", "emilia")
-beatrice = User("blog/user", "beatrice")
-
-post_one_reader = ObjectAndRelation(
-    namespace="blog/post",
-    object_id="1",
-    relation="reader",
+emilia = SubjectReference(
+    object=ObjectReference(
+        object_type="blog/user",
+        object_id="emilia",
+    )
 )
-post_one_writer = ObjectAndRelation(
-    namespace="blog/post",
-    object_id="1",
-    relation="writer",
+beatrice = SubjectReference(
+    object=ObjectReference(
+        object_type="blog/user",
+        object_id="beatrice",
+    )
 )
 
+post_one = ObjectReference(object_type="blog/post", object_id="1")
+
 client = Client(
     "grpc.authzed.com:443",
     bearer_token_credentials("t_your_token_here_1234567deadbeef"),
 )
 
-resp = client.Check(CheckRequest(user=emilia, test_userset=post_one_reader))
-assert resp.is_member
-resp = client.Check(CheckRequest(user=emilia, test_userset=post_one_writer))
-assert resp.is_member
-
-resp = client.Check(CheckRequest(user=beatrice, test_userset=post_one_reader))
-assert resp.is_member
-resp = client.Check(CheckRequest(user=beatrice, test_userset=post_one_writer))
-assert not resp.is_member
+resp = client.CheckPermission(
+    CheckPermissionRequest(
+        resource=post_one,
+        permission="reader",
```

Comment:
Use read and write permissions in the python checks

Reply:
i copied this from the repo 😬